rpcwebsocket.go

```diff
@@ -1845,7 +1847,7 @@ func (c *wsClient) serviceRequest(r *parsedRPCCmd) {
 	if ok {
 		result, err = wsHandler(c, r.params)
 	} else {
-		result, err = c.rpcServer.standardCmdResult(r, nil)
+		result, err = c.rpcServer.standardCmdResult(context.Background(), r)
```

Comment:
Same case here.  This really should be coming from further up the stack.  Should be `context.TODO()` for now.